go.mod

```diff
@@ -11,4 +11,5 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/sourcegraph/lsif-protocol v1.0.0
 	google.golang.org/protobuf v1.25.0 // indirect
+	gopkg.in/yaml.v2 v2.3.0 // indirect
```

Comment:
It's not clear to me why this line is added here but it got added from `go mod tidy` 🤷

Reply:
Yeah if sum didn't change that's fine. It was already a dep. I don't 100% grok the internals of how these files operate.